utils/funcexp/func_monthname.cpp

```diff
@@ -165,12 +165,20 @@ int64_t Func_monthname::getIntVal(rowgroup::Row& row, FunctionParm& parm, bool&
 
       break;
 
-    default: isNull = true; return -1;
+    default:
+      isNull = true;
+      return -1;
   }
 
   return -1;
 }
 
+int64_t Func_monthname::getIntVal(rowgroup::Row& row, FunctionParm& parm, bool& isNull,
+                                  execplan::CalpontSystemCatalog::ColType& op_ct)
+{
+  return 0;
```

Comment:
Looks suspicious.

Reply:
Still, MTR is running now. Regression is green, though.